src/EnergyPlus/SingleDuct.cc

```diff
@@ -2877,8 +2877,10 @@ void SingleDuctAirTerminal::SizeSys(EnergyPlusData &state)
 
             CheckZoneSizing(state, this->sysType, this->SysName);
 
-            MaxAirVolFlowRateDes = max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesCoolVolFlow,
-                                       state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesHeatVolFlow);
+            Real64 heatingMaxFlow = (this->DamperHeatingAction == Action::ReverseWithLimits)
+                                        ? state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesHeatVolFlowMax
+                                        : state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesHeatVolFlow;
+            MaxAirVolFlowRateDes = max(state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesCoolVolFlow, heatingMaxFlow);
```

Comment:
If ReverseWithLimits uses a different heating air flow rate from the zone design heating air flow rate then that flow rate should be used as the value for the VAV box heating air flow rate.

Reply:
Well, I was expecting DesHeatVolFlowMax to always be less than DesHeatVolFlow. For 5ZoneBoilerOutsideAirReset that's not the case. The diffs show that Zone 5 TU max air flow rate increased, I was only expecting decreases in max air flow rate for heating dominated zones. Zone cooling design air flow is greater than zone heating design air flow, but DesHeatVolFlowMax is larger than both of these.

![image](https://github.com/user-attachments/assets/cc35cd60-ec92-4651-a1cf-7e525a00e2c7)

![image](https://github.com/user-attachments/assets/eb7e6395-f186-4da7-aceb-db6f33528d7b)

    // set the zone maximum heating supply air flow rate. This will be used for autosizing VAV terminal unit
    // max heating flow rates
    zsFinalSizing.DesHeatVolFlowMax = max(zsFinalSizing.DesHeatMaxAirFlow,
                                          zsFinalSizing.DesHeatMaxAirFlow2,
                                          max(zsFinalSizing.DesCoolVolFlow, zsFinalSizing.DesHeatVolFlow) * zsFinalSizing.DesHeatMaxAirFlowFrac);

    zsFinalSizing.DesHeatMaxAirFlow2 = zsFinalSizing.DesHeatMaxAirFlowPerArea * floorArea * zoneMult;
                                                             = 0.00203 * 182.49 * 1 = 0.37082

    N8 , \field Maximum Flow Fraction During Reheat
         \type real
         \autosizable
         \default autosize
         \note Used only when Reheat Coil Object Type = Coil:Heating:Water and Damper Heating Action = ReverseWithLimits
         \note When autocalculating, the maximum flow fraction is set to the ratio of
         \note 0.002032 m3/s-m2 (0.4 cfm/sqft) multiplied by the zone floor area and the

Reply:
So maybe I should change this line and line 3224 to?:

    Real64 heatingMaxFlow = (DesHeatVolFlowMax > DesHeatVolFlow) ? state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesHeatVolFlow :
        state.dataSize->TermUnitFinalZoneSizing(state.dataSize->CurTermUnitSizingNum).DesHeatVolFlowMax);

Reply:
Based on this discussion I changed to treat DesHeatVolFlowMax as an upper limit. This should still correct the issue and result in fewer diffs.